zipkin-collector-service/config/collector-cassandra.scala

```diff
@@ -25,11 +25,18 @@ import org.twitter.zipkin.storage.cassandra.ZipkinRetryPolicy
 val contactPoints: Array[String] = sys.env.get("CASSANDRA_CONTACT_POINTS").getOrElse("localhost")
   .split(",")
 
-val cluster = Cluster.builder()
-  .addContactPoints(contactPoints:_*)
-  .withSocketOptions(new SocketOptions().setConnectTimeoutMillis(10000).setReadTimeoutMillis(20000))
-  .withRetryPolicy(ZipkinRetryPolicy.INSTANCE)
-  .build()
+val cassandraUser: String = sys.env.get("CASSANDRA_USER").getOrElse("")
+val cassandraPass: String = sys.env.get("CASSANDRA_PASS").getOrElse("")
+
+val clusterBuilder = if(cassandraUser == "" && cassandraPass == "")
+  Cluster.builder()
+else
+  Cluster.builder().withCredentials(cassandraUser, cassandraPass)
+
+val cluster = clusterBuilder.addContactPoints(contactPoints:_*)
+    .withSocketOptions(new SocketOptions().setConnectTimeoutMillis(10000).setReadTimeoutMillis(20000))
+    .withRetryPolicy(ZipkinRetryPolicy.INSTANCE)
+    .build()
```

Comment:
It looks like if we use the factory we would use it [here](https://github.com/openzipkin/zipkin/blob/master/zipkin-query-service/src/main/scala/com/twitter/zipkin/builder/QueryServiceBuilder.scala#L39)?  Maybe I could do that in another PR.

 or were you thinking something like:
```
object TestFactory extends App with CassandraSpanStoreFactory
cluster = TestFactory.createClusterBuilder()
```
Not sure if this is valid ^^^